app/models/concerns/push_updates_for_message.rb

```diff
@@ -1,18 +1,16 @@
 #
-# Updation of client store logic for messages is extracted here
```

Comment:
I've renamed this module from `UpdateClientStoreAndSendNotification` to `PushUpdatesForMessage` - it still includes the notification methods and I think that is fine too.